pkg/api/common_test.go

```diff
@@ -376,3 +368,8 @@ func callAPI(server *web.Mux, method, path string, body io.Reader, t *testing.T)
 	server.ServeHTTP(recorder, req)
 	return recorder
 }
+
+func mockRequestBody(v interface{}) io.ReadCloser {
+	b, _ := json.Marshal(v)
+	return io.NopCloser(bytes.NewReader(b))
+}
```

Comment:
Previously some handlers took a DTO object. Now all of them take a request context. Thus - DTOs should be mocked by faking a JSON body of the request. This function turns a DTO struct into a JSON Body (io.ReadCloser).